SoliditybasicsCourse/test/CourseToken.test.js

```diff
@@ -0,0 +1,41 @@
+const { expect, use, assert } = require("chai");
+const {
+  deployContract,
+  MockProvider,
+  BigNumber,
+  solidity,
+} = require("ethereum-waffle");
+const BasicContract = require("../build/RunContract.json");
+
+use(solidity);
+
+describe("Greeter", function () {
+  const provider = new MockProvider();
+  const [wallet, walletTo] = new MockProvider().getWallets();
+
+  let token;
+
+  beforeEach(async () => {
+    token = await deployContract(wallet, BasicContract, []);
+  });
+
+  it("Set new value", async () => {
+    expect(await token.setInterface(2, 2));
```

Comment:
This expect doesn't check anything.
```suggestion
    await token.setInterface(2, 2);
```